### ISSUE TITLE
Code not being loaded on patient_landing.php

### ISSUE THREAD

**Issue by kienmarkdo** [CSI2532-Databases-I-Group-8/Dental-Clinic-Management-System#7]
There is a strange issue where the HTML code below line 457 (Appointment Procedures) REFUSES to load on the page. I opened inspect element and cannot see the div attributes at ALL. Nothing is displayed. I initially thought there was a maximum number of rows allowed to be displayed in the container or something, but the HTML code is just not displayed at all.

![image](https://user-images.githubusercontent.com/67518620/163759283-bc941670-26c0-4388-a564-7ff87257320b.png)

![image](https://user-images.githubusercontent.com/67518620/163759330-1abc57ae-406b-44cc-a0e2-9ce685b2a318.png)

![image](https://user-images.githubusercontent.com/67518620/163759377-71bfede8-57f8-445e-9683-ccf693518b3a.png)

**Comment by kienmarkdo** [comment]
**TLDR Solution:** Load the page on the incognito browser next time to avoid cache problems.

Problem solved. I thought the problem was XAMPP but turns out we just needed to do a cache-refresh.

**Comment by kienmarkdo** [comment]
The exact same issue is happening again even on incognito mode. Nothing that is after Appointment Procedures is displayed.

**Comment by kienmarkdo** [comment]
Inconsistent and buggy. My guess is it's because of XAMPP.
